src/clockkit.cs

```diff
@@ -13,12 +13,42 @@
 using UIKit;
 
 namespace ClockKit {
+
+	[Watch (7,0), iOS (14, 0)]
+	[ErrorDomain ("CLKWatchFaceLibraryErrorDomain")]
+	[Native]
+	public enum CLKWatchFaceLibraryErrorCode : long {
+		NotFileURL = 1,
+		InvalidFile = 2,
+		PermissionDenied = 3,
+		FaceNotAvailable = 4,
+	}
 	
 	[BaseType (typeof (NSObject))]
 	interface CLKComplication : NSCopying {
 
 		[Export ("family")]
 		CLKComplicationFamily Family { get; }
+
+		[Watch (7, 0), NoiOS]
+		[Export ("identifier")]
+		string Identifier { get; }
+
+		[Watch (7, 0), NoiOS]
+		[Export ("userInfo"), NullAllowed]
+		NSDictionary UserInfo { get; }
+
+		[Watch (7, 0), NoiOS]
+		[Export ("userActivity"), NullAllowed]
+		NSUserActivity UserActivity { get; }
+
+		[Watch (7, 0), NoiOS]
+		[Field ("CLKLaunchedComplicationIdentifierKey")]
+		NSString LaunchedComplicationIdentifierKey { get; }
```

Comment:
strong dict over `UserInfo` ?

Reply:
As per docs the UserInfo is: 

> An optional dictionary that can be used to pass information back to your extension via CLKComplication.

Is a dictionary used by the extension and not something that is used by the API. But the key is definitely in the wrong place, should be in CLKLaunchOptionsKeys